master/getting-started/kubernetes/upgrade/downgrade.md

```diff
@@ -11,5 +11,56 @@ your cluster after running `calico-upgrade complete`, you should do so as soon
 as possible to avoid an outage. Any pods created after `calico-upgrade complete`
 and before downgrading will lose networking.
 
-_More docs on this coming soon!_
+## Downgrading a self-hosted installation
 
+1. If you have upgraded the {{site.prodname}} components after running
+   `calico-upgrade start` do the following steps. If you have not upgraded
+   the {{site.prodname}} components yet, skip to
+   [Aborting the upgrade](#aborting-the-upgrade).
+
+1. Use the following commands to initiate a downgrade of the {{site.prodname}} components.
+
+   ```
+   kubectl rollout undo ds/calico-node
```

Comment:
I think you need `-n kube-system` right?

Reply:
yep, pretty sure it will